app/src/main/java/fr/free/nrw/commons/contributions/ContributionsActivity.java

```diff
@@ -139,7 +140,11 @@ protected void onCreate(Bundle savedInstanceState) {
         requestAuthToken();
         initDrawer();
         setTitle(getString(R.string.title_activity_contributions));
-        setUploadCount();
+
+        if(!BuildConfig.FLAVOR.equalsIgnoreCase("beta")){
+            setUploadCount();
+        }
```

Comment:
I am not sure about this if statement since it should also work on Production flavor. Somehow, it works but I didn't convinced yet.

Reply:
Oh, there is "!". Sorry I was expecting something like BuildConfig.FLAVOR.equalsIgnoreCase("prod"):D